src/schedule/edf_schedule.c

```diff
@@ -22,6 +22,8 @@
 #include <stddef.h>
 #include <stdint.h>
 
+#include <sof/lib/cpu.h>
```

Comment:
Can you make this a new PR, there will be discussion on this that will block the others being merged.